### ISSUE TITLE
New Icon: Guild Wars 2

### ISSUE THREAD

**Issue by mepherion** [simple-icons/simple-icons#9522]
### Brand Name

Guild Wars 2

### Website

https://www.guildwars2.com/en

### Popularity Metric

Global Rank: 6,620
See https://www.similarweb.com/website/guildwars2.com

### Official Resources for Icon and Color

https://www.guildwars2.com/en/media/asset-kit/

### Contributing

No

**Comment by RatatoskrOfMana** [comment]
Hoping to see this added

**Comment by uncenter** [comment]
@RatatoskrOfMana feel free to read through https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md and make a contribution!

**Comment by runxel** [comment]
Apart from it's complexity I am pretty sure that we can not include this logo. The [TOS](https://www.arena.net/de/legal/content-terms-of-use) forbids this.

**Comment by uncenter** [comment]
I would agree that [those terms](https://www.arena.net/en/legal/content-terms-of-use) mean we can't include this.
